uhubctl.c

```diff
@@ -1209,7 +1233,30 @@ int main(int argc, char *argv[])
         exit(1);
     }
 
+#if defined(LIBUSB_API_VERSION) && (LIBUSB_API_VERSION >= 0x01000107) && \
+  (defined(__gnu_linux__) || defined(__linux__))
+    if (opt_sysdev) {
+        sys_fd = open(opt_sysdev, O_RDWR);
+        if (sys_fd < 0) {
+            fprintf(stderr, "Cannot open system node!\n");
+            rc = 1;
+            goto cleanup;
+        }
+        rc = libusb_wrap_sys_device(NULL, sys_fd, &sys_devh);
+        if (rc != 0) {
+            fprintf(stderr, "Cannot wrap system node!\n");
```

Comment:
Please change error message to be more descriptive. E.g. something like this: cannot use %s as USB hub device!

Reply:
Well, it is also important to also convey exactly what is going wrong, but I fully agree on adding the provided device name in the message.